lasy/optical_elements/optical_element.py

```diff
@@ -35,6 +35,9 @@ def amplitude_multiplier(self, x, y, omega, omega0):
         omega0: float
             Central angular frequency of the laser.
 
+        omega_0 : float (in rad/s)
+            Central angular frequency.
+
```

Comment:
`omega0` still appears above in the docstring, is this on purpose? Or a bug somewhere?
Also, the comment could clarify the difference for the two omegas, e.g.:
```suggestion
        omega_0 : float (in rad/s)
            Central angular frequency, as used for the definition of the laser envelope.

```